spec.go

```diff
@@ -0,0 +1,63 @@
+package specs
+
+// Spec is the base configuration for the container.  It specifies platform
+// independent configuration.
+type Spec struct {
+	// Version is the version of the specification that is supported.
+	Version string `json:"version"`
+	// Platform is the host information for OS and Arch.
+	Platform Platform `json:"platform"`
+	// Process is the container's main process.
+	Process Process `json:"process"`
+	// Root is the root information for the container's filesystem.
+	Root Root `json:"root"`
+	// Hostname is the containers host name.
+	Hostname string `json:"hostname"`
+	// Mounts profile configuration for adding mounts to the container's filesystem.
+	Mounts []Mount `json:"mounts"`
+}
+
+// Mount specifies a mount for a container.
+type Mount struct {
+	// Type specifies the mount kind.
+	Type string `json:"type"`
+	// Source specifies the source path of the mount.  In the case of bind mounts on
+	// linux based systems this would be the file on the host.
+	Source string `json:"source"`
+	// Destination is the path where the mount will be placed relative to the container's root.
+	Destination string `json:"destination"`
+	// Options are fstab style mount options.
+	Options string `json:"options"`
+}
+
+// Process contains information to start a specific application inside the container.
+type Process struct {
+	// Terminal creates an interactive terminal for the container.
+	Terminal bool `json:"terminal"`
+	// User specifies user information for the process.
+	User User `json:"user"`
+	// Args specifies the binary and arguments for the application to execute.
+	Args []string `json:"args"`
+	// Env populates the process environment for the process.
+	Env []string `json:"env"`
+	// Cwd is the current working directory for the process and must be
+	// relative to the container's root.
+	Cwd string `json:"cwd"`
+}
+
+// Root contains information to the container's root filesystem with the path on the host.
```

Comment:
s/information to/information about/ 
Also maybe just "Root container information about the container's root filesystem on the host" ?